.betterer.results

```diff
@@ -1,5 +1,5 @@
 // BETTERER RESULTS V2.
-//
+// 
```

Comment:
This appears each time I raise a PR related to the deprecated components. Should I merge it? It's just a blank space 🤷

Reply:
That is super weird. There wouldn't be much harm in merging it, but it might be removed in a random PR, which may be confusing.